cluster.yaml.example

```diff
@@ -5,6 +5,11 @@ externalAPIAddress: 10.0.0.1
 externalVPNDNSName: vpn.hosted.example.com
 externalVPNPort: 3118
 serviceCIDR: 172.31.0.0/16
+namedCerts:
+  - namedCertPrefix: cert1
```

Comment:
make the prefix include the path as well it needs to be the whole path
`"/etc/kubernetes/secret/cert1.crt"`

Reply:
it should ultimately be `/etc/kubernetes/secret/cert1` since that is where we will put these named certs for now